com.unity.render-pipelines.core/Editor/CoreEditorDrawers.cs

```diff
@@ -534,6 +537,20 @@ void IDrawer.Draw(TData data, Editor o)
             return FoldoutGroup(title, mask, state, options, null, null, contentDrawers);
         }
 
+        static string GetHelpURL<TEnum>(TEnum mask)
+            where TEnum : struct, IConvertible
+        {
+            var helpURLAttribute = (HelpURLAttribute)mask
+                .GetType()
+                .GetCustomAttributes(typeof(HelpURLAttribute), false)
+                .FirstOrDefault();
+
+            if (helpURLAttribute == null)
+                return string.Empty;
+
+            return $"{helpURLAttribute.URL}#{mask}";
```

Comment:
~~Is doc is ok to have this anchor in doc pages?~~
Well seams it was already the case